doc/sources/what-is-patching.rst

```diff
@@ -37,4 +37,12 @@ Patching
 
          python sklearnex.glob patch_sklearn
       
-      .. seealso:: :ref:`global_patching`
\ No newline at end of file
+      .. seealso:: :ref:`global_patching`
+   
+   how it works
```

Comment:
this is the syntax to introduce a new term in a glossary, meaning that you are introducing a "how it works" term, I don't think this is a good approach

you can make it a subsection, e.g.:

```
How patching works
-----------------------
```

or: introduce a definition of `dispatching` (the second sentence of the definition below), but the "how it works" still shouldn't be in the glossary